docs/packages/beluga

```diff
@@ -0,0 +1 @@
+../../beluga
```

Comment:
Note the symlinks. To keep package documentation as close as possible to package code, I had to pull in packages under the Sphinx source directory. I don't love it but some of the Sphinx extensions we use leave little margin of action.